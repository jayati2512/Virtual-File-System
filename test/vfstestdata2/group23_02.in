createvfs demo 100
mountvfs demo
mountvfs demo
makedir /home/
makedir /home/ a1
makedir /home/a1/ dir1
unmountvfs demo
makedir /home/a2/ dir2
mountvfs demo
makedir /home/a2/ dir2
listdir /home/ 0 /home/user/Desktop/out.txt
listdir /home/ 0 
listdir /home/ 0 /home/user/out.txt
unmountvfs demo
listdir /home/ 0 /home/user/out.txt
mountvfs demo
makedir /home/a2/dir3 
listdir /home/ 1 /home/user/out.txt
unmountvfs demo
