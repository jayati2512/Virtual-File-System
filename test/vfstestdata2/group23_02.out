createvfs_SUCCESS
mountvfs_SUCCESS
mountvfs_FAILURE: VFS_ALREADY_MOUNTED
makedir_FAILURE: VFS_INSUFFICIENT_ARGUMENTS
makedir_SUCCESS
makedir_SUCCESS
unmountvfs_SUCCESS
makedir_FAILURE: VFS_NOT_MOUNTED
mountvfs_SUCCESS
makedir_SUCCESS
listdir_FAILURE: CANNOT_FIND_SPECIFIED_PATH_OR_DIR
listdir_FAILURE: VFS_INSUFFICIENT_ARGUMENTS
listdir_FAILURE: CANNOT_FIND_SPECIFIED_PATH_OR_DIR
unmountvfs_SUCCESS
listdir_FAILURE: VFS_NOT_MOUNTED
mountvfs_SUCCESS
makedir_FAILURE: VFS_INSUFFICIENT_ARGUMENTS
listdir_FAILURE: CANNOT_FIND_SPECIFIED_PATH_OR_DIR
unmountvfs_SUCCESS
